### PR TITLE
feat(post-movies): adds a create locations endpoint

### PR DESCRIPTION
**Pull request by forbaokhanh**
WHAT: adds a create locations endpoint
WHY: completing my miyagi training

### CODE REVIEW

File: test/plugins/features/locations/controller.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    it('creates a location', () => {
+      const payload = { name: '1917 Delaware Street' };
+
+      return Controller.create(payload)
+      .then((location) => {
+        expect(location.get('name')).to.eql(payload.name);
+
+        return new Location({ id: location.id}).fetch();
+
```
Comment (itsachen) [inline test/plugins/features/locations/controller.test.js]: Nit: Remove newline

File: test/validators/locations/create.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+'use strict';
```
Comment (itsachen) [inline test/validators/locations/create.js:1]: Nit: Add a newline here

File: test/validators/locations/create.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+const Joi            = require('joi');
```
Comment (itsachen) [inline test/validators/locations/create.js]: Nit: You don't actually have to align the Joi require - we usually separate 3rd party requires from internal module requires with a newline. Aligning `=`'s is done separately between those two groups (3rd party/internal modules)

File: test/validators/locations/create.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      const payload = {};
+      const result = Joi.validate(payload, LocationCreateValidator);
+
+      expect(result.error.details[0].path).to.eql('name');
+      expect(result.error.details[0].type).to.eql('any.required');
+    });
+
+    it('is less than 255 characters', () => {
+      // create a test payload
+      const payload = {name : 'a'.repeat(256)};
```
Comment (itsachen) [inline test/validators/locations/create.js]: Nit: Pad the object literal with spaces.

**Review by itsachen — APPROVED** [review]
LGTM!

**Review by sjl2 — COMMENTED** [review]
nit: In this case, you're adding a create locations endpoint, the commit message for this would be more clear like `feat(locations): add create endpoint`

File: test/validators/locations/create.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    it('is required', () => {
+      const payload = {};
+      const result = Joi.validate(payload, LocationCreateValidator);
+
+      expect(result.error.details[0].path).to.eql('name');
+      expect(result.error.details[0].type).to.eql('any.required');
+    });
+
+    it('is less than 255 characters', () => {
+      // create a test payload
```
Comment (sjl2) [inline test/validators/locations/create.js]: nit: you can probably remove this comment now.

**Review by sjl2 — APPROVED** [review]
L G T M! couple nits but good to go!

**Pull request merged**